Makefile

```diff
@@ -2,15 +2,15 @@
 #   rsync
 #   wget or curl
 #
-JRUBY_VERSION=1.7.11
-ELASTICSEARCH_VERSION=1.1.1
+JRUBY_VERSION=1.7.17
```

Comment:
I've tried to make it work with .19 without success, the last working version for 1.4.2 for me is .18.